### DISCUSSION TITLE
How can i get the length of the scorin period for each matchup period?

### DISCUSSION THREAD

**Discussion by JLMad** [cwendt94/espn-api#274] [Q&A]
This is so cool! I am very new to all of this and am using espn-api to try and work on some intro to coding. I would like to gather  team by team categorical stats for a given day. I think this information is there in the box score lineups, but I am a little confused on how to correctly input the matchup period and scoring period, or better yet how to loop through an entire season. Our league has a two week stretch for all stat weekend so the scoring period per week is variable. Can you point me to where this information is kept?

**Comment by cwendt94** [comment]
Which sport is this for? Scoring period and matchup period get a little tricky depending on how the league is setup and what scoring method they are using.

**Reply by JLMad** [reply]
This is for basketball. Thanks!

**Reply by cwendt94** [reply]
For basketball I currently have not found a way using ESPN API data to get the known scoring periods in a matchup. For some reason they don't have a mapping of matchups and scoring periods in their API. So trying to find data by day you will have to find the scoring period through ESPN Website. It shows the scoring period in the url when you go to fantasycast and change the different days in the matchup.

The [wiki](https://github.com/cwendt94/espn-api/wiki/League-Class-Basketball) for `box_scores` has some information if you want data for the whole matchup or individual days.
